### ISSUE TITLE
Add pagination / time filter to recap

### ISSUE THREAD

**Issue by Fort-TM** [drekdrek/grinding-stats#54]
## Suggestion

Some people's recaps can have thousands of maps, so I think it would help if:

* The recap could have pagination, instead of infinite scrolling
* The user could choose to only load the maps from the last day / week / month / etc.

For the latter, the plugin could keep a JSON with the map UUIDs and when they were last played, and only load the files that match those UUIDs in that time frame. Something like

```json
[
    ["OAmhby4s9zR4IMzUu8Zkh1sxd3c", 1731947838],
    ["Ye6btWgKsS2M4vCqGsL8COTOUoh", 1729438863]
]
```
(example uses epoch time)

## Credits

Time filter idea originally suggested by Tarpor on the Openplanet Discord server ([Link](https://discord.com/channels/276076890714800129/1047245343923900437/1306034952030261359))

**Comment by drekdrek** [comment]
we could also use the last modified time of the file to detect when it was last played, that is how the current implementation works in the recap display

**Comment by Fort-TM** [comment]
Yeah, that also works and would avoid requiring another file. Good idea 👍
